src/metrics.go

```diff
@@ -165,6 +165,11 @@ func getMetricsData(sample *metric.MetricSet) error {
 	if err != nil {
 		return err
 	}
+	// If is not a successful status code 2xx.
+	if resp.StatusCode/100 != 2 {
```

Comment:
why not just `!= 200` by dividing you are adding an additional computation that is not required.